src/blas/backends/rocblas/rocblas_level1.cpp

```diff
@@ -971,6 +971,9 @@ cl::sycl::event sdsdot(cl::sycl::queue &queue, int64_t n, float sb, const float
         });
     });
     done.wait();
+#if(defined(ENABLE_ROCBLAS_BACKEND) && !defined(__HIPSYCL__))
+    queue.wait();
```

Comment:
Why did you need to add this? Wouldn't done.wait() be enough?

Reply:
Thanks for the explanation. Do you know when will the LLVM compiler will be updated? Could you please paste the PR for LLVM here?